_potd_floorsets/101.md

```diff
@@ -86,6 +86,11 @@ boss_job_specifics:
       - '5m45s with no offensive pomanders (6.10)'
     notes:
       - 'Easy with no pomanders'
+  WHM:
+    timing:
+      - "5m30s with no offensive pomanders (7.16)
```

Comment:
```suggestion
      - "5m30s with no offensive pomanders (7.16)"
```

Reply:
My copypasta laziness is revealed :( Fixed all these.